composer.json

```diff
@@ -11,14 +11,48 @@
             "role": "Company"
         }
     ],
+    "repositories": [
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/chameleon-shop"
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/chameleon-shop-theme-bundle"
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/chameleon-base",
+            "options": {
+                "versions": {
+                    "chameleon-system/chameleon-base": "7.2.x-dev"
+                }
+            }
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/sanitycheck"
+        },
+        {
+            "type": "path",
+            "url": "packages/chameleon-system/sanitycheck-bundle"
+        }
+    ],
     "require": {
-        "chameleon-system/chameleon-shop": "~7.1.0",
-        "chameleon-system/chameleon-shop-theme-bundle": "~7.1.0",
+        "chameleon-system/chameleon-base": "7.2.x-dev as 7.1.99",
+        "chameleon-system/chameleon-shop": "7.2.x-dev as 7.1.99",
+        "chameleon-system/chameleon-shop-theme-bundle": "7.2.x-dev as 7.1.99",
         "incenteev/composer-parameter-handler": "~2.0",
-        "sensio/framework-extra-bundle": "*"
+        "symfony/security-bundle": "^5.4"
```

Comment:
This comes with chameleon core, doesn't it?

Reply:
true